utils/clflags.ml

```diff
@@ -67,6 +67,7 @@ and use_threads = ref false             (* -thread *)
 and use_vmthreads = ref false           (* -vmthread *)
 and noassert = ref false                (* -noassert *)
 and verbose = ref false                 (* -verbose *)
+and noversion = ref false                 (* -q *)
```

Comment:
Damien Doligez (2016/05/04 08:00 -0700):
> You should fix the comment.

Oops sorry, fixed!
And sorry for the typo in branch name.